Model/IO/Import/ANSYS/Ansys.cpp

```diff
@@ -0,0 +1,115 @@
+#include "Ansys.h"
+using namespace std;
+
+MeshIS::Model::Common::CMR   Ansys::Import(const string & absolute_file_path)
+{
+    MeshIS::Model::Common::CMR;
+    fstream file;
+    file.open(absolute_file_path);
+    skipFirstlineFile(file);
+    setSizeNodeElement(file);
+    tabNode=new Point3D[sizeNode];
+    tabElement= new Point5D[sizeElement];
+    int index=0,counter=getHowMany(file);
+    importNodes(index,counter,tabNode,file);
+
+    index=counter,counter=getHowMany(file);
+    importNodes(index,counter,tabNode,file);
+
+    index=0,counter=getHowMany(file);
+    importElements(index,counter,tabElement,file);
+
+    index=counter,counter=getHowMany(file);
+    importElements(index,counter,tabElement,file);
+    MeshIS::Model::Common::CMR result=savetoCmrObject();
+    file.close();
+    return result;
+}
+void Ansys::skipFirstlineFile(fstream &file) {
+    string z;
+    int i=0;
+    do
+    {
+        getline(file,z);
+        i++;
+    }
+    while(i<8);
+}
+int  Ansys::getHowMany(fstream &file) {
+    string z;
+    int i=0;
+    do {
+        file >> z;
+        i++;
+    }
+    while(i!=3);
+    int start = std::stoi(z, 0, 16);
+    file>>z;
+    int end= std::stoi(z, 0, 16);;
+    int num=end-start+1;
+    getline(file,z);
+    return num;
+}
+void Ansys::importNodes(int index,int HowMany,Point3D* tab,fstream & file){
```

Comment:
This is imporant stuff: what is a meaning of the pointer here? Owning? Not-owning, container?
What is a rationale to use naked pointer here instead of unique_ptr, shared_ptr, vector or whatever?